example/server.dart

```diff
@@ -60,6 +61,7 @@ ArgParser argsParser() {
   parser.addOption('host', abbr: 'h', defaultsTo: 'localhost');
 
   parser.addOption('port', abbr: 'p', defaultsTo: '8080');
+  parser.addOption('standalone', abbr: 's', defaultsTo: 'false');
```

Comment:
A suggestion: If you use `addFlag` instead of `addOption` you get the boolification automatically:
```dart
parser.addFlag('standalone', defaultsTo: false);
```